src/course-outline/CourseOutline.test.jsx

```diff
@@ -113,6 +114,30 @@ describe('<CourseOutline />', () => {
     expect(await findByText(messages.alertSuccessDescription.defaultMessage)).toBeInTheDocument();
   });
 
+  it('check video sharing option udpates correctly', async () => {
+    const { findByTestId } = render(<RootWrapper />);
+
+    axiosMock
+      .onPost(getEnableHighlightsEmailsApiUrl(courseId), {
```

Comment:
@Cup0fCoffee Yes, I am aware of it. It is being updated as part of https://github.com/openedx/frontend-app-course-authoring/pull/769 [here](https://github.com/openedx/frontend-app-course-authoring/pull/769/files#diff-d748dd7f9c7e80062e9a5490ad598972a03929b4c02d8abe86f62adbebd85adeR22) so I left it as is to avoid conflicts. Once https://github.com/openedx/frontend-app-course-authoring/pull/769 is merged, I'll update its usage. OR I can update it now and resolve conflicts later on.

Reply:
I think it makes sense to update it now because this is where you are introducing its use for something other than highlight emails.